index.js

```diff
@@ -378,14 +377,14 @@ function evaluate(feature, user) {
     }
   }  
 
-  for (var i = 0; i < feature.variations.length; i ++) {
+  for (i = 0; i < feature.variations.length; i ++) {
```

Comment:
Maybe I should just hoist the declaration of `var` to the start of the function?